src/main/kotlin/com/bardsoftware/papeeria/backend/cosmas/CosmasGoogleCloudService.kt

```diff
@@ -65,7 +65,10 @@ class CosmasGoogleCloudService(private val bucketName: String,
             if (fileVersion != null) {
                 project[request.fileId] = fileVersion.toBuilder().addPatches(request.patch).build()
             } else {
-                project[request.fileId] = CosmasProto.FileVersion.newBuilder().addPatches(request.patch).build()
+                project[request.fileId] = CosmasProto.FileVersion.newBuilder().
+                        setContent(ByteString.copyFrom("".toByteArray())).
```

Comment:
`ByteString.EMPTY` ? Isn't it the default by any chance?

Reply:
ok, it's `ByteString.EMPTY` by default